build.mk

```diff
@@ -65,23 +65,13 @@ native-deb: # HOST   == riscv64
 	find $(DESTDIR) -exec touch -d "$(KERNEL_TIMESTAMP)" {} \;
 	SOURCE_DATE_EPOCH="1" dpkg-deb -Zxz --root-owner-group --build $(DESTDIR) $(NATIVE_DEB_FILENAME)
 
-# configure riscv-pk
+# build linux w/ opensbi
 # ------------------------------------------------------------------------------
-$(RISCV_PK_BUILD_DIR)/Makefile: $(LINUX_DIR)/vmlinux $(LINUX_DIR)/.config
-	@mkdir -p $(RISCV_PK_BUILD_DIR)
-	cd $(RISCV_PK_BUILD_DIR) && ../configure \
-		--with-payload=$(abspath $<) \
-		--disable-fp-emulation \
-		--host=$(TOOLCHAIN_PREFIX)
-
-# build linux w/ bbl
-# ------------------------------------------------------------------------------
-$(LINUX): $(RISCV_PK_DIR)/build/Makefile $(LINUX_DIR)/vmlinux
-	mkdir -p artifacts
-	$(MAKE) $(JOBS) -rC $(RISCV_PK_BUILD_DIR) bbl
-	$(TOOLCHAIN_PREFIX)-objcopy \
-		-O binary $(RISCV_PK_BUILD_DIR)/bbl $@
-	truncate -s %4096 $@
+OPENSBI_OPTS=$(JOBS) FW_PAYLOAD_PATH=$(abspath $(IMAGE)) PLATFORM=cartesi CROSS_COMPILE=$(TOOLCHAIN_PREFIX)-
```

Comment:
We should also add `FW_OPTIONS=0x01` to disable OpenSBI debug prints, which is annoying.